data/json/item_groups.json

```diff
@@ -340,7 +340,6 @@
       ["gum", 30],
       ["caff_gum", 3],
       ["flask_hip", 5],
-      ["sm_extinguisher", 10],
```

Comment:
Was it intentional to remove `sm_extinguisher` from `living_room`?

Reply:
Yes, it is double of `sm_extinguisher` on line 315.